src/instructlab/model/chat.py

```diff
@@ -635,11 +646,11 @@ def start_prompt(
             self.multiline_mode = 0
             self.multiline = not self.multiline
 
-        # Optional parameters
-        create_params = {}
-        if self.greedy_mode:
+        # Temperature parameters
```

Comment:
It looks like the default temperature if `self.greedy_mode` was `False` here before? https://platform.openai.com/docs/api-reference/chat/create#chat-create-temperature

@RobotSail what would you think about making the default for temperature `None`? I think we'd want some number to be the default so that the user always knows what the temperature is?

Reply:
@alimaredia For temperature we should never use `None`. It should be an explicit float between 0 & 1.